libraries/AP_RPM/RPM_Backend.h

```diff
@@ -32,7 +32,7 @@ class AP_RPM_Backend
     virtual void update() = 0;
 
     int8_t get_pin(void) const {
-        if (state.instance > 1) {
```

Comment:
Pre exsisting, but surely this is paranoid/indicates something is seriously misconfigured? How does `state.instance` ever get configured to something outside of `RPM_MAX_INSTANCES` ?

Reply:
it can't unless you add an extra backend or change the backend index. The index is assigned by a loop that goes around RPM_MAX_INSTANCES times. using the define is better than a magic number

Reply:
Oh I'm fine with the define, I'm just not really sure that the check is actually required at all. It's implying what should be an unreachable condition.